bpf/bpf_host.c

```diff
@@ -1334,8 +1350,12 @@ int cil_to_netdev(struct __ctx_buff *ctx __maybe_unused)
 	__u16 proto = 0;
 #endif
 
-	if (magic == MARK_MAGIC_HOST || magic == MARK_MAGIC_OVERLAY)
+	if (magic == MARK_MAGIC_HOST || magic == MARK_MAGIC_OVERLAY || ctx_mark_is_wireguard(ctx))
```

Comment:
PR 32871 is marked as partial, but I believe the commit is fully picked. Is this expected?

Reply:
no you're correct! I misremembered how the change back then was structured.